srcpkgs/adwaita-icon-theme/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'adwaita-icon-theme'
 pkgname=adwaita-icon-theme
-version=41.0
+version=42.0
 revision=1
 build_style=gnu-configure
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config gtk+3"
```

Comment:
This creates a cycle - gtk+3 - adwaita-icon-theme - gtk+3

Reply:
idk how you would resolve that? It still needs `gtk-encode-symbolic-svg` for the symbolic icons?